crates/solvers/src/domain/solver/baseline.rs

```diff
@@ -214,6 +234,24 @@ impl Inner {
             })
             .filter(|r| !r.sell.amount.is_zero() && !r.buy.amount.is_zero())
     }
+
+    fn native_price_request(&self, order: UserOrder) -> Request {
+        let sell = eth::Asset {
+            token: order.get().sell.token,
+            amount: eth::U256::exp10(18),
```

Comment:
Should it be configurable the same way the native price estimator has it?

Reply:
Not necessarily configurable but it should follow the same semantics. Instead of selling 10^18 of a potential shitcoin with unknown decimals for weth, we want to "buy 1 WETH with up to max_uint256 sell token" and use the price thereof.

Reply:
This? https://github.com/cowprotocol/services/blob/main/crates/driver/src/domain/quote.rs#L191-L202

Reply:
Yup, 0.1WETH is used: https://github.com/cowprotocol/infrastructure/blob/staging/services/Pulumi.mainnet-services-prod.yaml#L98

Will adjust the code, thanks!

Reply:
Actually that probably poses a slight problem because xdai and mainnet have different values. So it would have to be configurable in the baseline estimator as well. :/
xdai uses 1dai (10**18) which is also a bit strange given that 0.1 WETH >> 1dai. 😅